file_store/src/reward_manifest.rs

```diff
@@ -1,12 +1,27 @@
 use crate::{error::DecodeError, traits::MsgDecode, Error};
 use chrono::{DateTime, TimeZone, Utc};
 use helium_proto as proto;
+use rust_decimal::Decimal;
 
 #[derive(Clone, Debug)]
 pub struct RewardManifest {
     pub written_files: Vec<String>,
     pub start_timestamp: DateTime<Utc>,
     pub end_timestamp: DateTime<Utc>,
+    pub reward_data: Option<RewardData>,
+}
+
+#[derive(Clone, Debug)]
+pub enum RewardData {
+    MobileRewardData {
+        poc_bones_per_coverage_point: Decimal,
+        boosted_poc_bones_per_coverage_point: Decimal,
```

Comment:
```suggestion
        poc_bones_per_reward_share: Decimal,
        boosted_poc_bones_per_reward_share: Decimal,
```
These should be representing portions of a reward share like the IoT reward data.

"Coverage Points" has covered many meaning in the past. Ideally it refers to points rewarded _only_ for covering hexes. When combined with Multipliers they become a radios "Reward Shares".

This change is also taking place over in [radio_reward_v2 proto](https://github.com/helium/proto/pull/407/files#diff-41a424c2a5f8e8fc2a265baae2b8a8f277ea1d7d21a1a4bb73a2ab391c971e6eR490-R495).